x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/hooks/setup_technology.ts

```diff
@@ -121,13 +123,14 @@ export function useSetupTechnology({
   useEffect(() => {
     const shouldBeDefault =
       isAgentlessEnabled &&
+      !disableAgentless &&
```

Comment:
`disableAgentless` is a bit confusing since we also have `isAgentlessEnabled` already in this context. What about a more explicit name, for instance `hideAgentlessSelector` or something like this?